contrib/win-installer/podman-msihooks/check.c

```diff
@@ -64,3 +79,52 @@ BOOL isWSLEnabled() {
 
 	return exitCode == 0;
 }
+
+BOOL isHyperVEnabled() {
+	/*
+	 * Checks if the Windows service `vmcompute` is running to
```

Comment:
Looking for service names is a good idea around the dism perm issues, but vmcompute is also used by WSL, so we need another way to detect it

Reply:
You are right, thank you. 

I have done some tests and it looks like the service `vmms` (Hyper-V Virtual Machine Management service) is not running when `wsl` is installed and is actually running when Hyper-V is installed. Would that work in your opinion?

![image](https://github.com/containers/podman/assets/606959/30e08373-5463-4ab2-a3dd-9c2b698b2478)

![image](https://github.com/containers/podman/assets/606959/51b40edb-0010-4f38-94e4-2e6677e774b3)

Reply:
yes that sounds like it should work